vt/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.5.4'
+__version__ = '0.6.1'
```

Comment:
This should be marked as a minor, set it to 0.5.5 please

Reply:
Last tag is 0.6.0, is it possible that this file was not updated?

Reply:
Hm, let me check, I just realized we had some pending PRs, maybe version tagging was not updated at some point

Reply:
Ok, seems we have a discordance between file tag and GitHub tag. Let's go to v0.6.1, nothing to change in this PR XD